recipes/opensubdiv/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
You probably need to set some minimum compiler requirements in a validate method to check https://github.com/PixarAnimationStudios/OpenSubdiv/blob/ff76e0f2dc9c9202b7d2f965f264bfd6f41866d5/opensubdiv/CMakeLists.txt#L87

This is my boiler plate https://github.com/conan-io/conan/issues/8002

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-is-a-toolscheck_min_cppstd-call-not-enough

Reply:
I'm not sure how to deal with this. OpenSubdiv does not mention any minimum C++ standard or compiler versions. They only require C++14 when building with Metal support, so I'm not sure what I could check for. As far as I tested, `set(CMAKE_CXX_STANDARD 14) `seems not to be needed, I think I added it accidently by copy-paste.

So my solution would be to check for C++14 when building with Metal support  and remove the `set(CMAKE_CXX_STANDARD 14)` line?

Reply:
In the conan recipe, you should add the boilerplate from https://github.com/conan-io/conan/issues/8002

You can adjust the compiler requirement for `options` like this example https://github.com/conan-io/conan-center-index/blob/fd308ea48a5ccffe971146a1d414bdc82ac3ea1b/recipes/stlab/all/conanfile.py#L94-L95